src/layout/Summary/SummaryComponent.tsx

```diff
@@ -100,21 +100,23 @@ export const SummaryComponent = React.forwardRef(function SummaryComponent(
     : (summaryNode?.id ?? targetNode?.id ?? 'unknown');
 
   return (
-    <Grid
+    <Flex
       ref={ref}
-      item={true}
-      xs={displayGrid?.xs || 12}
-      sm={displayGrid?.sm || false}
-      md={displayGrid?.md || false}
-      lg={displayGrid?.lg || false}
-      xl={displayGrid?.xl || false}
+      item
+      size={{
+        xs: displayGrid?.xs ?? 12,
+        sm: displayGrid?.sm,
+        md: displayGrid?.md,
+        lg: displayGrid?.lg,
+        xl: displayGrid?.xl,
+      }}
```

Comment:
Doesn't these do the same thing? :thinking: 
```suggestion
      size={displayGrid}
```

Reply:
No, the difference is that the fallback for xs on 12 is not applied.

Reply:
So if you don't specify a size for `xs`, what does it do when you shrink it down? :thinking: I would imagine 12 is a good default.

Reply:
If you dont specify size for xs you dont get fluid sizing. This is the default behavior of MUI. it would still be 12 as a default but not fluid sizing. We could rewrite this to be default fluid 12, but it would require a rewrite css of many of our components. The priority in this PR is just to replicate what we already had. Many possibilities to improve our grid system now that we own it ourselves.

Reply:
I see! :raised_hands: